client/src/App.jsx

```diff
@@ -33,20 +33,37 @@ export default class App extends Component {
   };
 
   componentDidMount() {
-    socket.emit('getmails');
+    socket.on('error', data => console.log(data));
+    socket.on('request getmails', ()=> { 
+      socket.emit('getmails', {range:'1:5'});
```

Comment:
why do we specify the range here?

Reply:
to get a specific number of messages until the user scrolls down or click on next for example.